src/plugins/developer-settings/index.tsx

```diff
@@ -137,6 +138,15 @@ function setupDebugger({ patcher, cleanup }: PluginContextFor<typeof plugin, 'Af
     )
 
     globalThis.reload = () => BundleUpdaterManager.reload()
+    globalThis.getRidOfPlugin = id => {
```

Comment:
We also need to get rid of `pluginStates[id]`, my suggestion is wait for the upcoming change I'm about to push. I'll rebase the changes